tools/mxnet/mxnet2ncnn.cpp

```diff
@@ -838,6 +847,9 @@ int main(int argc, char** argv)
         else if (n.op == "BatchNorm")
         {
             float eps = n.attr("eps");
+            if (fabs(eps) < 1e-6) {
```

Comment:
why eps should be changed here

Reply:
the default eps in mxnet is 1e-3. in some version, it is not writed in symbol.json

like this: 

```
{
      "op": "null",
      "name": "batchnorm18_gamma",
      "inputs": []
    },
    {
      "op": "null",
      "name": "batchnorm18_beta",
      "inputs": []
    },
    {
      "op": "null",
      "name": "batchnorm18_moving_mean",
      "attr": {"__init__": "[\"zero\", {}]"},
      "inputs": []
    },
    {
      "op": "null",
      "name": "batchnorm18_moving_var",
      "attr": {"__init__": "[\"one\", {}]"},
      "inputs": []
    },
    {
      "op": "BatchNorm",
      "name": "batchnorm18",
      "inputs": [[165, 0, 0], [166, 0, 0], [167, 0, 0], [168, 0, 1], [169, 0, 1]]
    },
```

Reply:
ok, but it could be checked using
```n.has_attr("eps")```

Reply:
updated